### ISSUE TITLE
[feature] generate automatically a `.gitignore` in `.vscode/perl-lang` to avoid git trace it

### ISSUE THREAD

**Issue by Freed-Wu** [richterger/Perl-LanguageServer#177]
TIA!

**Comment by azazar** [comment]
`.vscode/perl-lang` is really annoying. It's created even for non-perl workspaces.

**Comment by yzazik** [comment]
This is still rather individual, but you can always configure git to have a global gitignore file:
```bash
git config --global core.excludesfile <git-ignore-global-file>
```

**Comment by azazar** [comment]
> This is still rather individual, but you can always configure git to have a global gitignore file:
> 
> git config --global core.excludesfile <git-ignore-global-file>

No. It's a bug that annoys user and serves no purpose. It should be recognized as a bug.

**Comment by yzazik** [comment]
> > This is still rather individual, but you can always configure git to have a global gitignore file:
> > git config --global core.excludesfile
> 
> No. It's a bug that annoys user and serves no purpose. It should be recognized as a bug.

Agree that it's a bug creating it for non-perl workspaces (whatever that means).
And it's a bug creating it when Disable Cache is on.
But these are rather non-git issues, not sure how adding .gitignore would solve either.
